src/storage-client/src/controller/collection_mgmt.rs

```diff
@@ -20,11 +20,17 @@ use mz_persist_types::Codec64;
 use mz_repr::{Diff, GlobalId, Row, TimestampManipulation};
 use timely::progress::Timestamp;
 use tokio::sync::{mpsc, oneshot};
+use tokio::time::{Duration, Instant};
 use tracing::debug;
 
 use crate::client::TimestamplessUpdate;
 use crate::controller::{persist_handles, StorageError};
 
+// Note(parkmycar): The capacity here was chosen randomly.
```

Comment:
super nit (i know this was just code movement): s/randomly/arbitrarily/ :)

Reply:
done :)